src/test/scala/uk/gov/hmcts/ccd/corecasedata/scenarios/ValidateCaseDetails.scala

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.ccd.corecasedata.scenarios
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.ccd.util.{CcdTokenGenerator, PerformanceTestsConfig}
+
+import scala.concurrent.duration._
+
+object ValidateCaseDetails extends PerformanceTestsConfig {
+
+ // val EventId = "applyForGrant"
+  val EventId = "CREATE"
+  val CreateCaseUrl = caseDataUrl(config.getString("validateCaseUrl"))
+  val CreateCaseTokenUrl = s"${CreateCaseUrl.replaceAll("cases", "")}event-triggers/$EventId/token"
+  println("create case url: " + CreateCaseUrl)
+  println("create case token url: " + CreateCaseTokenUrl)
+
+  private val url: String = config.getString("caseDataUrl") + "/" + config.getString("validateCaseDetails")
+  println("Retrieving validateCaseDetails URL : " + url)
```

Comment:
what does this mean? we don't retrieve urls

Reply:
Amended. it's just printing URL as output.